serverless-iam-policy.json

```
@@ -113,7 +113,8 @@
         "logs:CreateLogGroup",
         "logs:DescribeLogGroups",
         "logs:DescribeLogStreams",
-        "logs:PutRetentionPolicy"
+        "logs:PutRetentionPolicy",
+        "logs:TagResource"
```

You may also want to add `UntagResource`... at least, I did for `dynamodb:TagResource`, and I'm guessing it's similar for `logs:TagResource`:
```suggestion
        "logs:TagResource",
        "logs:UntagResource"
```

---

Sorry, I just (re-)noticed that this is a release PR. Feel free to do this in a separate feature branch PR (if desired).